packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 47.3.0 - October 24, 2019
+
+## Added
```

Comment:
FYI this PR #1730 might still make it into tomorrow's release. I'm just waiting on some cli tests.

Reply:
This may also make it into tomorrow's release. #1741